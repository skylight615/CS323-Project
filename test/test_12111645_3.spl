int sum(int x, int y)
{
  int res = 1 + c;
  return x;
}
int test_2_r02()
{
  int a1, a2;
  float b = a1;
  return b;
}