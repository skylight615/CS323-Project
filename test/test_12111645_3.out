Error type 1 at Lind 3: c is used without a definition
Error type 5 at Line 9: unmatching type on both sides of assignment
Error type 8 at Line 10: incompatiable return type
